sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -1027,4 +1026,79 @@ class MapboxMapTest {
       styleObserver.setLoadStyleListener(options, any(), any())
     }
   }
+}
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
```

Comment:
is it necessary to use `Robolectric` here?

Reply:
yes, to use shadow class to bypass the Logger that goes through gl-native

Reply:
okay